database/sinkdb/state.go

```diff
@@ -90,6 +90,13 @@ func (s *state) RestoreSnapshot(data []byte, index uint64) error {
 	s.peers = snapshot.Peers
 	s.state = snapshot.State
 	s.version = snapshot.Version
+
```

Comment:
Added the map initialization since it was creating errors when we manually triggered snapshots: now that we've gotten rid of that, should we leave this in?